Auto update failing for nushell

Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the nushell package, but it failed.

Here's the output of the update script:
<details>
<summary>Show log</summary>
<pre lang="bash">
INFO: Updating nushell [Current version: 0.67.0]
INFO: package being updated to 0.68.0.

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100  224k    0  224k    0     0  1272k      0 --:--:-- --:--:-- --:--:-- 1272k
100 2324k    0 2324k    0     0  6510k      0 --:--:-- --:--:-- --:--:-- 11.3M

You are about to commit these changes:

--------------------
diff --git a/packages/nushell/build.sh b/packages/nushell/build.sh
index 5f06bf8e0..d2fb46948 100644
--- a/packages/nushell/build.sh
+++ b/packages/nushell/build.sh
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://www.nushell.sh
 TERMUX_PKG_DESCRIPTION="A new type of shell operating on structured data"
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="0.67.0"
+TERMUX_PKG_VERSION="0.68.0"
 TERMUX_PKG_SRCURL=https://github.com/nushell/nushell/archive/$TERMUX_PKG_VERSION.tar.gz
-TERMUX_PKG_SHA256=a8c4c092087d035805781dff11000f2b7c4a44ae487b013f9d0a59c4dc610546
+TERMUX_PKG_SHA256=c170ce2a0fa931194c8169585608747fea0cbf7c957f67c6e2fd54c0c5071c64
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_DEPENDS="openssl, zlib"
 TERMUX_PKG_AUTO_UPDATE=true
--------------------

nushell: update to 0.68.0


Not committing to Git!

INFO: Trying to build package.
Running container 'termux-package-builder' from image 'ghcr.io/termux/package-builder'...
termux - building nushell for arch aarch64...
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13133  100 13133    0     0   116k      0 --:--:-- --:--:-- --:--:--  117k
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   2861      0 --:--:-- --:--:-- --:--:--  2872
gpg: Signature made Tue 06 Sep 2022 06:32:39 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13319  100 13319    0     0  44950      0 --:--:-- --:--:-- --:--:-- 44845
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   2754      0 --:--:-- --:--:-- --:--:--  2758
gpg: Signature made Sun 04 Sep 2022 06:32:25 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13119  100 13119    0     0  46235      0 --:--:-- --:--:-- --:--:-- 46356
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   2680      0 --:--:-- --:--:-- --:--:--  2678
gpg: Signature made Fri 02 Sep 2022 11:16:28 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading dependency ca-certificates@1:2022.07.19 if necessary...
Skipping already built dependency ca-certificates@1:2022.07.19
Downloading dependency ca-certificates-java@1:2022.07.19 if necessary...
Skipping already built dependency ca-certificates-java@1:2022.07.19
Downloading dependency zlib@1.2.12-1 if necessary...
Skipping already built dependency zlib@1.2.12-1
Downloading dependency openssl@3.0.5 if necessary...
Skipping already built dependency openssl@3.0.5
Downloading dependency openssl-tool@3.0.5 if necessary...
Skipping already built dependency openssl-tool@3.0.5
Downloading https://github.com/nushell/nushell/archive/0.68.0.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 2324k  100 2324k    0     0   9.9M      0 --:--:-- --:--:-- --:--:--  9.9M
Applying patch: user_home_dir.patch
info: downloading installer
info: profile set to 'default'
info: default host triple is x86_64-unknown-linux-gnu
warning: Updating existing toolchain, profile choice will be ignored
info: syncing channel updates for '1.62.1-x86_64-unknown-linux-gnu'
info: default toolchain set to '1.62.1-x86_64-unknown-linux-gnu'

  1.62.1-x86_64-unknown-linux-gnu unchanged - rustc 1.62.1 (e092d0b6b 2022-07-16)


Rust is installed now. Great!

To get started you may need to restart your current shell.
This would reload your PATH environment variable to include
Cargo's bin directory ($HOME/.cargo/bin).

To configure your current shell, run:
source "$HOME/.cargo/env"
info: syncing channel updates for '1.61.0-x86_64-unknown-linux-gnu'
info: latest update on 2022-05-19, rust version 1.61.0 (fe5b13d68 2022-05-18)
info: downloading component 'cargo'
info: downloading component 'clippy'
info: downloading component 'rust-docs'
info: downloading component 'rust-std'
info: downloading component 'rustc'
info: downloading component 'rustfmt'
info: installing component 'cargo'
info: installing component 'clippy'
info: installing component 'rust-docs'
info: installing component 'rust-std'
info: installing component 'rustc'
info: installing component 'rustfmt'
info: downloading component 'rust-std' for 'aarch64-linux-android'
info: installing component 'rust-std' for 'aarch64-linux-android'
 Downloading crates ...
  Downloaded pkg-config v0.3.25
  Downloaded futures-io v0.3.21
  Downloaded num_threads v0.1.6
  Downloaded rayon-core v1.9.3
  Downloaded tempfile v3.3.0
  Downloaded time-macros v0.2.4
  Downloaded itoa v1.0.2
  Downloaded time v0.3.13
  Downloaded openssl-sys v0.9.75
  Downloaded lz4 v1.23.3
  Downloaded integer-encoding v3.0.4
  Downloaded parquet-format-async-temp v0.3.1
  Downloaded snap v1.0.5
  Downloaded rusty-fork v0.3.0
  Downloaded remove_dir_all v0.5.3
  Downloaded proc-macro-error-attr v1.0.4
  Downloaded futures-sink v0.3.21
  Downloaded fastrand v1.8.0
  Downloaded pin-project-lite v0.2.9
  Downloaded thiserror-impl v1.0.31
  Downloaded futures-task v0.3.21
  Downloaded proc-macro-error v1.0.4
  Downloaded libc v0.2.126
  Downloaded pin-utils v0.1.0
  Downloaded futures-util v0.3.21
  Downloaded autocfg v1.1.0
  Downloaded crossbeam-epoch v0.9.10
  Downloaded quote v1.0.20
  Downloaded unicode-segmentation v1.9.0
  Downloaded signal-hook-registry v1.4.0
  Downloaded futures v0.3.21
  Downloaded glob v0.3.0
  Downloaded num_cpus v1.13.1
  Downloaded calamine v0.18.0
  Downloaded rustversion v1.0.8
  Downloaded rstest_macros v0.14.0
  Downloaded hashlink v0.8.0
  Downloaded utf8parse v0.2.0
  Downloaded strip-ansi-escapes v0.1.1
  Downloaded simplelog v0.12.0
  Downloaded futures-executor v0.3.21
  Downloaded zeroize v1.5.7
  Downloaded lexical-write-integer v0.8.5
  Downloaded lock_api v0.4.7
  Downloaded vte v0.10.1
  Downloaded crossbeam-deque v0.8.2
  Downloaded is-root v0.1.2
  Downloaded predicates-core v1.0.3
  Downloaded predicates-tree v1.0.5
  Downloaded fallible-streaming-iterator v0.1.9
  Downloaded comfy-table v5.0.1
  Downloaded vcpkg v0.2.15
  Downloaded phf_codegen v0.10.0
  Downloaded lz4-sys v1.9.4
  Downloaded console v0.15.0
  Downloaded dirs v4.0.0
  Downloaded joinery v2.1.0
  Downloaded json-deserializer v0.3.1
  Downloaded filesize v0.2.0
  Downloaded ctrlc v3.2.2
  Downloaded is_ci v1.1.1
  Downloaded supports-hyperlinks v1.2.0
  Downloaded supports-unicode v1.0.2
  Downloaded signal-hook-mio v0.2.3
  Downloaded assert_cmd v2.0.4
  Downloaded hamcrest2 v0.3.0
  Downloaded dirs-sys v0.3.7
  Downloaded fs_extra v1.2.0
  Downloaded block-buffer v0.10.2
  Downloaded diff v0.1.13
  Downloaded notify v4.0.17
  Downloaded utf-8 v0.7.6
  Downloaded strum_macros v0.24.2
  Downloaded uuid v1.1.2
  Downloaded pure-rust-locales v0.5.6
  Downloaded fuzzy-matcher v0.3.7
  Downloaded quick-xml v0.19.0
  Downloaded phf_shared v0.8.0
  Downloaded phf_codegen v0.8.0
  Downloaded phf_shared v0.11.0
  Downloaded strum v0.23.0
  Downloaded foreign_vec v0.1.0
  Downloaded typed-arena v1.7.0
  Downloaded termtree v0.2.4
  Downloaded is_executable v1.0.1
  Downloaded phf_generator v0.11.0
  Downloaded multiversion v0.6.1
  Downloaded difflib v0.4.0
  Downloaded quick-error v1.2.3
  Downloaded phf_macros v0.8.0
  Downloaded memmap2 v0.5.5
  Downloaded planus v0.3.1
  Downloaded serde_urlencoded v0.7.1
  Downloaded htmlescape v0.3.1
  Downloaded servo_arc v0.1.1
  Downloaded crossterm v0.23.2
  Downloaded brotli-decompressor v2.3.2
  Downloaded phf v0.11.0
  Downloaded lexical-parse-float v0.8.5
  Downloaded foreign-types-shared v0.1.1
  Downloaded smawk v0.3.1
  Downloaded linked-hash-map v0.5.6
  Downloaded toml v0.5.9
  Downloaded chrono-tz-build v0.0.3
  Downloaded phf_codegen v0.11.0
  Downloaded scraper v0.13.0
  Downloaded thread_local v1.1.4
  Downloaded dialoguer v0.9.0
  Downloaded lexical-core v0.8.5
  Downloaded chrono-humanize v0.2.1
  Downloaded byte-unit v4.0.14
  Downloaded users v0.10.0
  Downloaded rand_distr v0.4.3
  Downloaded parse-zoneinfo v0.3.0
  Downloaded fancy-regex v0.10.0
  Downloaded void v1.0.2
  Downloaded meval v0.2.0
  Downloaded filetime v0.2.17
  Downloaded polars-lazy v0.23.2
  Downloaded polars-arrow v0.23.2
  Downloaded bytemuck v1.11.0
  Downloaded ppv-lite86 v0.2.16
  Downloaded pathdiff v0.2.1
  Downloaded sxd-xpath v0.4.2
  Downloaded peresil v0.3.0
  Downloaded sxd-document v0.3.2
  Downloaded chrono-tz v0.6.3
  Downloaded heapless v0.7.15
  Downloaded gimli v0.26.2
  Downloaded convert_case v0.4.0
  Downloaded markup5ever v0.11.0
  Downloaded html5ever v0.26.0
  Downloaded nom v1.2.4
  Downloaded hyper v0.14.20
  Downloaded phf_generator v0.10.0
  Downloaded string_cache v0.8.4
  Downloaded string_cache_codegen v0.5.2
  Downloaded codepage v0.1.1
  Downloaded rust_decimal v1.25.0
  Downloaded semver-parser v0.10.2
  Downloaded polars-utils v0.23.2
  Downloaded alphanumeric-sort v1.4.4
  Downloaded polars v0.23.2
  Downloaded dtoa-short v0.3.3
  Downloaded sqlparser v0.16.0
  Downloaded http v0.2.8
  Downloaded dtoa v0.4.8
  Downloaded phf_generator v0.8.0
  Downloaded env_logger v0.8.4
  Downloaded criterion-plot v0.4.5
  Downloaded tendril v0.4.3
  Downloaded socket2 v0.4.4
  Downloaded cssparser-macros v0.6.0
  Downloaded selectors v0.22.0
  Downloaded semver v0.11.0
  Downloaded cssparser v0.27.2
  Downloaded hyper-tls v0.5.0
  Downloaded quick-xml v0.23.0
  Downloaded rand_pcg v0.2.1
  Downloaded tower-service v0.3.2
  Downloaded proptest v1.0.0
  Downloaded smartstring v1.0.1
  Downloaded pest v2.1.3
  Downloaded getopts v0.2.21
  Downloaded zstd-safe v5.0.2+zstd.1.5.2
  Downloaded quickcheck_macros v1.0.0
  Downloaded derive_more v0.99.17
  Downloaded tracing-core v0.1.28
  Downloaded zstd-sys v2.0.1+zstd.1.5.2
  Downloaded mac v0.1.1
  Downloaded alloc-stdlib v0.2.1
  Downloaded libssh2-sys v0.2.23
  Downloaded fxhash v0.2.1
  Downloaded brotli v3.3.4
  Downloaded unicode-normalization v0.1.21
  Downloaded csv v1.1.6
  Downloaded tokio-util v0.7.3
  Downloaded ego-tree v0.6.2
  Downloaded ucd-trie v0.1.4
  Downloaded futf v0.1.5
  Downloaded h2 v0.3.13
  Downloaded libz-sys v1.1.8
  Downloaded parquet2 v0.14.1
  Downloaded simdutf8 v0.1.4
  Downloaded result v1.0.0
  Downloaded zstd v0.11.2+zstd.1.5.2
  Downloaded encoding_rs v0.8.31
  Downloaded gjson v0.8.1
  Downloaded openssl-src v111.22.0+1.1.1q
  Downloaded tokio v1.20.1
  Downloaded rust-embed v6.4.0
  Downloaded crypto-common v0.1.6
  Downloaded native-tls v0.2.10
  Downloaded num-rational v0.4.1
  Downloaded precomputed-hash v0.1.1
  Downloaded Inflector v0.11.4
  Downloaded lexical-write-float v0.8.5
  Downloaded openssl-probe v0.1.5
  Downloaded digest v0.9.0
  Downloaded quick-error v2.0.1
  Downloaded httparse v1.7.1
  Downloaded generic-array v0.14.5
  Downloaded tracing v0.1.35
  Downloaded xmlparser v0.13.3
  Downloaded users v0.11.0
  Downloaded array-init-cursor v0.2.0
  Downloaded sha2 v0.9.9
  Downloaded httpdate v1.0.2
  Downloaded vec1 v1.8.0
  Downloaded new_debug_unreachable v1.0.4
  Downloaded clap v2.34.0
  Downloaded unicode-bidi v0.3.8
  Downloaded pori v0.0.0
  Downloaded rand v0.7.3
  Downloaded reqwest v0.11.11
  Downloaded rust-embed-impl v6.2.0
  Downloaded block-buffer v0.9.0
  Downloaded sysinfo v0.25.2
  Downloaded roxmltree v0.14.1
  Downloaded criterion v0.3.6
  Downloaded csv-core v0.1.10
  Downloaded quickcheck v1.0.3
  Downloaded streaming-iterator v0.1.6
  Downloaded wax v0.5.0
  Downloaded textwrap v0.11.0
  Downloaded cast v0.3.0
  Downloaded opaque-debug v0.3.0
  Downloaded http-body v0.4.5
  Downloaded foreign-types v0.3.2
  Downloaded umask v2.0.0
  Downloaded want v0.3.0
  Downloaded num-complex v0.4.2
  Downloaded tz-rs v0.6.11
  Downloaded try-lock v0.2.3
  Downloaded ipnet v2.5.0
  Downloaded mime v0.3.16
  Downloaded tokio-native-tls v0.3.0
  Downloaded typenum v1.15.0
  Downloaded siphasher v0.3.10
  Downloaded which v4.3.0
  Downloaded sha2 v0.10.2
  Downloaded bytes v1.2.0
  Downloaded serde_cbor v0.11.2
  Downloaded zip v0.5.13
  Downloaded phf v0.10.1
  Downloaded rmp-serde v1.1.0
  Downloaded rmp v0.8.11
  Downloaded phf_shared v0.10.0
  Downloaded tzdb v0.3.1
  Downloaded rmpv v1.0.0
  Downloaded rust-embed-utils v7.2.0
  Downloaded digest v0.10.3
  Downloaded paste v1.0.8
  Downloaded is_debug v1.0.1
  Downloaded const_fn v0.4.9
  Downloaded tempdir v0.3.7
  Downloaded hash32 v0.2.1
  Downloaded libsqlite3-sys v0.25.1
  Downloaded byte-order v0.3.0
  Downloaded polars-core v0.23.2
  Downloaded stable_deref_trait v1.2.0
  Downloaded crc32fast v1.3.2
  Downloaded predicates v2.1.1
  Downloaded const_format v0.2.26
  Downloaded plotters v0.3.3
  Downloaded const_format_proc_macros v0.2.22
  Downloaded ctor v0.1.22
  Downloaded rand v0.8.5
  Downloaded nix v0.24.2
  Downloaded byteorder v1.4.3
  Downloaded oorandom v11.1.3
  Downloaded plotters-svg v0.3.3
  Downloaded half v1.8.2
  Downloaded miette v4.7.1
  Downloaded utf8-width v0.1.6
  Downloaded tinytemplate v1.2.1
  Downloaded procfs v0.14.0
  Downloaded tabled v0.8.0
  Downloaded hex v0.4.3
  Downloaded ansitok v0.1.0
  Downloaded inventory v0.2.3
  Downloaded rand v0.4.6
  Downloaded miette v5.1.1
  Downloaded num-complex v0.2.4
  Downloaded num v0.2.1
  Downloaded ghost v0.1.5
  Downloaded erased-serde v0.3.21
  Downloaded num-rational v0.2.4
  Downloaded ansi-str v0.3.0
  Downloaded bytecount v0.6.3
  Downloaded papergrid v0.5.1
  Downloaded doc-comment v0.3.3
  Downloaded cpufeatures v0.2.2
  Downloaded minimal-lexical v0.2.1
  Downloaded tabled_derive v0.4.0
  Downloaded uncased v0.9.7
  Downloaded backtrace v0.3.66
  Downloaded nom v7.1.1
  Downloaded num-bigint v0.2.6
  Downloaded addr2line v0.17.0
  Downloaded getset v0.1.2
  Downloaded rustc-demangle v0.1.21
  Downloaded num-format v0.4.0
  Downloaded sys-locale v0.2.1
  Downloaded num-iter v0.1.43
  Downloaded pwd v1.4.0
  Downloaded typetag-impl v0.1.8
  Downloaded typetag v0.1.8
  Downloaded indexmap v1.9.1
  Downloaded num-bigint v0.4.3
  Downloaded bit-vec v0.6.3
  Downloaded bit-set v0.5.3
  Downloaded openssl-macros v0.1.0
  Downloaded arrow2 v0.13.1
  Downloaded nodrop v0.1.14
  Downloaded polars-ops v0.23.2
  Downloaded shadow-rs v0.16.1
  Downloaded terminal_size v0.2.1
  Downloaded titlecase v2.0.0
  Downloaded arrayvec v0.4.12
  Downloaded owo-colors v3.4.0
  Downloaded strum_macros v0.23.1
  Downloaded hash_hasher v2.0.3
  Downloaded async-stream v0.3.3
  Downloaded streaming-decompression v0.1.0
  Downloaded lscolors v0.12.0
  Downloaded serde_ini v0.2.0
  Downloaded dunce v1.0.2
  Downloaded rand_xorshift v0.3.0
  Downloaded md-5 v0.10.1
  Downloaded multiversion-macros v0.6.1
  Downloaded unsafe-libyaml v0.2.2
  Downloaded serde_yaml v0.9.4
  Downloaded byte-slice-cast v1.2.1
  Downloaded lexical-parse-integer v0.8.6
  Downloaded lexical-util v0.8.5
  Downloaded miette-derive v5.1.1
  Downloaded alloc-no-stdlib v2.0.3
  Downloaded wait-timeout v0.2.0
  Downloaded async-stream-impl v0.3.3
  Downloaded miette-derive v4.7.1
  Downloaded crunchy v0.2.2
  Downloaded strength_reduce v0.2.3
  Downloaded indent_write v2.2.0
  Downloaded tinyvec v1.6.0
  Downloaded thin-slice v0.1.1
  Downloaded eml-parser v0.1.3
  Downloaded async-trait v0.1.56
  Downloaded plotters-backend v0.3.4
  Downloaded crossterm v0.24.0
  Downloaded rusqlite v0.28.0
  Downloaded dtparse v1.3.0
  Downloaded miniz_oxide v0.5.3
  Downloaded vte_generate_state_changes v0.1.1
  Downloaded rand_chacha v0.3.1
  Downloaded rand_core v0.6.3
  Downloaded object v0.29.0
  Downloaded unicode-linebreak v0.1.2
  Downloaded openssl v0.10.41
  Downloaded num-integer v0.1.45
  Downloaded rayon v1.5.3
  Downloaded version_check v0.9.4
  Downloaded serde_json v1.0.82
  Downloaded unicode-ident v1.0.2
  Downloaded num v0.4.0
  Downloaded supports-color v1.3.0
  Downloaded anyhow v1.0.58
  Downloaded libm v0.2.2
  Downloaded arrayvec v0.5.2
  Downloaded semver v1.0.12
  Downloaded futures-macro v0.3.21
  Downloaded futures-core v0.3.21
  Downloaded slab v0.4.7
  Downloaded io-lifetimes v0.7.2
  Downloaded overload v0.1.1
  Downloaded nu-ansi-term v0.46.0
  Downloaded crossbeam-channel v0.5.6
  Downloaded memchr v2.5.0
  Downloaded parking_lot v0.12.1
  Downloaded bytemuck_derive v1.1.1
  Downloaded pretty_assertions v1.2.1
  Downloaded reedline v0.11.0
  Downloaded either v1.7.0
  Downloaded ryu v1.0.10
  Downloaded strum v0.24.1
  Downloaded iana-time-zone v0.1.47
  Downloaded num-traits v0.2.15
  Downloaded time v0.1.44
  Downloaded chrono v0.4.22
  Downloaded mio v0.8.4
  Downloaded ical v0.7.0
  Downloaded polars-io v0.23.2
  Downloaded polars-time v0.23.2
  Downloaded itertools v0.10.3
  Downloaded getrandom v0.2.7
  Downloaded ahash v0.7.6
  Downloaded crossbeam-utils v0.8.11
  Downloaded cc v1.0.73
  Downloaded thiserror v1.0.31
  Downloaded powierza-coefficient v1.0.1
  Downloaded brownstone v3.0.0
  Downloaded nom-supreme v0.8.0
  Downloaded heck v0.4.0
  Downloaded rustix v0.35.7
  Downloaded heck v0.3.3
  Downloaded static_assertions v1.1.0
  Downloaded android_system_properties v0.1.5
  Downloaded unicode-xid v0.2.3
  Downloaded arrayvec v0.7.2
  Downloaded fd-lock v3.0.6
  Downloaded fallible-iterator v0.2.0
  Downloaded gethostname v0.2.3
  Downloaded phf v0.8.0
  Downloaded rand_chacha v0.2.2
  Downloaded getrandom v0.1.16
  Downloaded rand_core v0.5.1
  Downloaded errno v0.2.8
  Downloaded rstest v0.15.0
  Downloaded arrow-format v0.7.0
  Downloaded serial_test v0.8.0
  Downloaded serial_test_derive v0.8.0
  Downloaded lexical v6.1.1
  Downloaded dyn-clone v1.0.8
  Downloaded syn v1.0.98
  Downloaded bitpacking v0.8.4
  Downloaded memoffset v0.6.5
  Downloaded rustc_version v0.4.0
  Downloaded hashbrown v0.12.3
  Downloaded futures-channel v0.3.21
  Downloaded proc-macro2 v1.0.42
  Downloaded signal-hook v0.3.14
  Downloaded log v0.4.17
  Downloaded termcolor v1.1.3
  Downloaded smallvec v1.9.0
  Downloaded scopeguard v1.1.0
  Downloaded parking_lot_core v0.9.3
info: downloading installer
info: profile set to 'default'
info: default host triple is x86_64-unknown-linux-gnu
warning: Updating existing toolchain, profile choice will be ignored
info: syncing channel updates for '1.62.1-x86_64-unknown-linux-gnu'

info: default toolchain set to '1.62.1-x86_64-unknown-linux-gnu'
  1.62.1-x86_64-unknown-linux-gnu unchanged - rustc 1.62.1 (e092d0b6b 2022-07-16)


Rust is installed now. Great!

To get started you may need to restart your current shell.
This would reload your PATH environment variable to include
Cargo's bin directory ($HOME/.cargo/bin).

To configure your current shell, run:
source "$HOME/.cargo/env"
info: component 'rust-std' for target 'aarch64-linux-android' is up to date
  Installing nu v0.68.0 (/home/builder/.termux-build/nushell/src)
    Updating crates.io index
warning: package `atomic-polyfill v0.1.8` in Cargo.lock is yanked in registry `crates-io`, consider running without --locked
warning: package `cpufeatures v0.2.2` in Cargo.lock is yanked in registry `crates-io`, consider running without --locked
 Downloading crates ...
  Downloaded inotify-sys v0.1.5
  Downloaded mio-extras v2.0.6
  Downloaded inotify v0.7.1
  Downloaded lazycell v1.3.0
  Downloaded mio v0.6.23
  Downloaded linux-raw-sys v0.0.46
  Downloaded trash v2.1.5
  Downloaded net2 v0.2.37
  Downloaded cfg-if v0.1.10
  Downloaded iovec v0.1.4
   Compiling libc v0.2.126
   Compiling autocfg v1.1.0
   Compiling proc-macro2 v1.0.42
   Compiling quote v1.0.20
   Compiling unicode-ident v1.0.2
   Compiling syn v1.0.98
   Compiling cfg-if v1.0.0
   Compiling version_check v0.9.4
   Compiling memchr v2.5.0
   Compiling once_cell v1.13.1
   Compiling serde_derive v1.0.144
   Compiling serde v1.0.144
   Compiling libm v0.2.2
   Compiling log v0.4.17
   Compiling scopeguard v1.1.0
   Compiling crossbeam-utils v0.8.11
   Compiling bitflags v1.3.2
   Compiling either v1.7.0
   Compiling rayon-core v1.9.3
   Compiling itoa v1.0.2
   Compiling smallvec v1.9.0
   Compiling adler v1.0.2
   Compiling unicode-width v0.1.9
   Compiling parking_lot_core v0.9.3
   Compiling regex-syntax v0.6.27
   Compiling futures-core v0.3.21
   Compiling signal-hook v0.3.14
   Compiling pin-project-lite v0.2.9
   Compiling pure-rust-locales v0.5.6
   Compiling typenum v1.15.0
   Compiling futures-sink v0.3.21
   Compiling futures-task v0.3.21
   Compiling futures-channel v0.3.21
   Compiling ryu v1.0.10
   Compiling lazy_static v1.4.0
   Compiling futures-util v0.3.21
   Compiling futures-io v0.3.21
   Compiling pin-utils v0.1.0
   Compiling serde_json v1.0.82
   Compiling pkg-config v0.3.25
   Compiling static_assertions v1.1.0
   Compiling crc32fast v1.3.2
   Compiling siphasher v0.3.10
   Compiling rustversion v1.0.8
   Compiling fallible-streaming-iterator v0.1.9
   Compiling async-trait v0.1.56
   Compiling fnv v1.0.7
   Compiling alloc-no-stdlib v2.0.3
   Compiling zstd-safe v5.0.2+zstd.1.5.2
   Compiling utf8parse v0.2.0
   Compiling arrayvec v0.4.12
   Compiling arrayvec v0.5.2
   Compiling crunchy v0.2.2
   Compiling nodrop v0.1.14
   Compiling itoa v0.4.8
   Compiling minimal-lexical v0.2.1
   Compiling snap v1.0.5
   Compiling ansi_term v0.12.1
   Compiling bit-vec v0.6.3
   Compiling ppv-lite86 v0.2.16
   Compiling const_fn v0.4.9
   Compiling gimli v0.26.2
   Compiling erased-serde v0.3.21
   Compiling byteorder v1.4.3
   Compiling array-init-cursor v0.2.0
   Compiling is_ci v1.1.1
   Compiling unicode-segmentation v1.9.0
   Compiling base64 v0.13.0
   Compiling dunce v1.0.2
   Compiling bytes v1.2.0
   Compiling smawk v0.3.1
   Compiling rustc-demangle v0.1.21
   Compiling io-lifetimes v0.7.2
   Compiling anyhow v1.0.58
   Compiling owo-colors v3.4.0
   Compiling strength_reduce v0.2.3
   Compiling streaming-iterator v0.1.6
   Compiling simdutf8 v0.1.4
   Compiling rustix v0.35.7
   Compiling json-deserializer v0.3.1
   Compiling hash_hasher v2.0.3
   Compiling utf8-width v0.1.6
   Compiling foreign_vec v0.1.0
   Compiling unicode-xid v0.2.3
   Compiling dyn-clone v1.0.8
   Compiling time-macros v0.2.4
   Compiling heck v0.4.0
   Compiling strum v0.23.0
   Compiling vcpkg v0.2.15
   Compiling openssl v0.10.41
   Compiling foreign-types-shared v0.1.1
   Compiling overload v0.1.1
   Compiling paste v1.0.8
   Compiling encoding_rs v0.8.31
   Compiling sysinfo v0.25.2
   Compiling native-tls v0.2.10
   Compiling same-file v1.0.6
   Compiling bytecount v0.6.3
   Compiling httparse v1.7.1
   Compiling nu-glob v0.68.0 (/home/builder/.termux-build/nushell/src/crates/nu-glob)
   Compiling arrayvec v0.7.2
   Compiling matches v0.1.9
   Compiling tinyvec_macros v0.1.0
   Compiling opaque-debug v0.3.0
   Compiling byte-slice-cast v1.2.1
   Compiling percent-encoding v2.1.0
   Compiling iana-time-zone v0.1.47
   Compiling cpufeatures v0.2.2
   Compiling openssl-probe v0.1.5
   Compiling try-lock v0.2.3
   Compiling num_threads v0.1.6
   Compiling tower-service v0.3.2
   Compiling procfs v0.14.0
   Compiling rust_decimal v1.25.0
   Compiling regex-automata v0.1.10
   Compiling unicode-bidi v0.3.8
   Compiling remove_dir_all v0.5.3
   Compiling byte-order v0.3.0
   Compiling is_debug v1.0.1
   Compiling fallible-iterator v0.2.0
   Compiling httpdate v1.0.2
   Compiling fastrand v1.8.0
   Compiling hex v0.4.3
   Compiling indent_write v2.2.0
   Compiling strum v0.24.1
   Compiling glob v0.3.0
   Compiling joinery v2.1.0
   Compiling zeroize v1.5.7
   Compiling result v1.0.0
   Compiling vec1 v1.8.0
   Compiling nom v1.2.4
   Compiling ipnet v2.5.0
   Compiling void v1.0.2
   Compiling unsafe-libyaml v0.2.2
   Compiling mime v0.3.16
   Compiling xmlparser v0.13.3
   Compiling pathdiff v0.2.1
   Compiling filesize v0.2.0
   Compiling alphanumeric-sort v1.4.4
   Compiling powierza-coefficient v1.0.1
   Compiling fs_extra v1.2.0
   Compiling umask v2.0.0
   Compiling bytesize v1.1.0
   Compiling is_executable v1.0.1
   Compiling termcolor v1.1.3
   Compiling nu v0.68.0 (/home/builder/.termux-build/nushell/src)
   Compiling htmlescape v0.3.1
   Compiling num-traits v0.2.15
   Compiling memoffset v0.6.5
   Compiling num-integer v0.1.45
   Compiling crossbeam-epoch v0.9.10
   Compiling rayon v1.5.3
   Compiling lock_api v0.4.7
   Compiling indexmap v1.9.1
   Compiling slab v0.4.7
   Compiling num-iter v0.1.43
   Compiling num-bigint v0.4.3
   Compiling num-rational v0.4.1
   Compiling tokio v1.20.1
   Compiling num-bigint v0.2.6
   Compiling num-rational v0.2.4
   Compiling num-complex v0.2.4
   Compiling ahash v0.7.6
   Compiling generic-array v0.14.5
   Compiling uncased v0.9.7
   Compiling proc-macro-error-attr v1.0.4
   Compiling proc-macro-error v1.0.4
   Compiling smartstring v1.0.1
   Compiling tracing-core v0.1.28
   Compiling thread_local v1.1.4
   Compiling itertools v0.10.3
   Compiling miniz_oxide v0.5.3
   Compiling lexical-util v0.8.5
   Compiling phf_shared v0.10.0
   Compiling streaming-decompression v0.1.0
   Compiling alloc-stdlib v0.2.1
   Compiling bit-set v0.5.3
   Compiling planus v0.3.1
   Compiling heck v0.3.3
   Compiling http v0.2.8
   Compiling addr2line v0.17.0
   Compiling byte-unit v4.0.14
   Compiling foreign-types v0.3.2
   Compiling nu-ansi-term v0.46.0
   Compiling walkdir v2.3.2
   Compiling brownstone v3.0.0
   Compiling tinyvec v1.6.0
   Compiling form_urlencoded v1.0.1
   Compiling meval v0.2.0
   Compiling roxmltree v0.14.1
   Compiling fuzzy-matcher v0.3.7
   Compiling tracing v0.1.35
   Compiling phf v0.10.1
   Compiling lexical-parse-integer v0.8.6
   Compiling lexical-write-integer v0.8.5
   Compiling brotli-decompressor v2.3.2
   Compiling http-body v0.4.5
   Compiling unicode-normalization v0.1.21
   Compiling num_cpus v1.13.1
   Compiling getrandom v0.2.7
   Compiling android_system_properties v0.1.5
   Compiling time v0.1.44
   Compiling signal-hook-registry v1.4.0
   Compiling atty v0.2.14
   Compiling terminal_size v0.1.17
   Compiling dirs-sys-next v0.1.2
   Compiling socket2 v0.4.4
   Compiling errno v0.2.8
   Compiling dirs-sys v0.3.7
   Compiling memmap2 v0.5.5
   Compiling time v0.3.13
   Compiling tempfile v3.3.0
   Compiling gethostname v0.2.3
   Compiling filetime v0.2.17
   Compiling which v4.3.0
   Compiling jobserver v0.1.24
   Compiling aho-corasick v0.7.18
   Compiling nom v7.1.1
   Compiling object v0.29.0
   Compiling csv-core v0.1.10
   Compiling quick-xml v0.23.0
   Compiling mio v0.8.4
   Compiling want v0.3.0
   Compiling users v0.10.0
   Compiling users v0.11.0
   Compiling crossbeam-channel v0.5.6
   Compiling flate2 v1.0.24
   Compiling lexical-parse-float v0.8.5
   Compiling lexical-write-float v0.8.5
   Compiling bitpacking v0.8.4
   Compiling brotli v3.3.4
   Compiling num-format v0.4.0
   Compiling tz-rs v0.6.11
   Compiling quick-xml v0.19.0
   Compiling codepage v0.1.1
   Compiling idna v0.2.3
   Compiling rand_core v0.6.3
   Compiling uuid v1.1.2
   Compiling supports-hyperlinks v1.2.0
   Compiling supports-color v1.3.0
   Compiling supports-unicode v1.0.2
   Compiling dirs-next v2.0.0
   Compiling dirs v4.0.0
   Compiling notify v4.0.17
   Compiling simplelog v0.12.0
   Compiling cc v1.0.73
   Compiling nix v0.24.2
   Compiling parking_lot v0.12.1
   Compiling phf_shared v0.11.0
   Compiling vte_generate_state_changes v0.1.1
   Compiling const_format_proc_macros v0.2.22
   Compiling regex v1.6.0
   Compiling ansitok v0.1.0
   Compiling nom-supreme v0.8.0
   Compiling pori v0.0.0
   Compiling num-complex v0.4.2
   Compiling rmp v0.8.11
   Compiling is-root v0.1.2
   Compiling block-buffer v0.9.0
   Compiling block-buffer v0.10.2
   Compiling digest v0.9.0
   Compiling crypto-common v0.1.6
   Compiling lexical-core v0.8.5
   Compiling tzdb v0.3.1
   Compiling url v2.2.2
   Compiling rand_chacha v0.3.1
   Compiling signal-hook-mio v0.2.3
   Compiling fd-lock v3.0.6
   Compiling terminal_size v0.2.1
   Compiling crossbeam-deque v0.8.2
   Compiling zstd-sys v2.0.1+zstd.1.5.2
   Compiling lz4-sys v1.9.4
   Compiling sys-locale v0.2.1
   Compiling backtrace v0.3.66
   Compiling openssl-sys v0.9.75
   Compiling libsqlite3-sys v0.25.1
   Compiling phf v0.11.0
   Compiling ctrlc v3.2.2
   Compiling vte v0.10.1
   Compiling const_format v0.2.26
   Compiling fancy-regex v0.10.0
   Compiling console v0.15.0
   Compiling eml-parser v0.1.3
   Compiling Inflector v0.11.4
   Compiling titlecase v2.0.0
   Compiling thiserror-impl v1.0.31
   Compiling futures-macro v0.3.21
   Compiling async-stream-impl v0.3.3
   Compiling ctor v0.1.22
   Compiling multiversion-macros v0.6.1
   Compiling ghost v0.1.5
   Compiling bytemuck_derive v1.1.1
   Compiling typetag-impl v0.1.8
   Compiling miette-derive v5.1.1
   Compiling strum_macros v0.23.1
   Compiling openssl-macros v0.1.0
   Compiling strum_macros v0.24.2
   Compiling miette-derive v4.7.1
   Compiling ansi-str v0.3.0
   Compiling parse-zoneinfo v0.3.0
   Compiling unicode-linebreak v0.1.2
   Compiling rmpv v1.0.0
   Compiling tokio-util v0.7.3
   Compiling sha2 v0.9.9
   Compiling digest v0.10.3
   Compiling lexical v6.1.1
   Compiling rand v0.8.5
   Compiling crossterm v0.23.2
   Compiling tabled_derive v0.4.0
   Compiling getset v0.1.2
   Compiling strip-ansi-escapes v0.1.1
   Compiling shadow-rs v0.16.1
   Compiling dialoguer v0.9.0
   Compiling thiserror v1.0.31
   Compiling async-stream v0.3.3
   Compiling inventory v0.2.3
   Compiling bytemuck v1.11.0
   Compiling multiversion v0.6.1
   Compiling rust-embed-utils v7.2.0
   Compiling md-5 v0.10.1
   Compiling sha2 v0.10.2
   Compiling comfy-table v5.0.1
   Compiling rand_distr v0.4.3
   Compiling nu-pretty-hex v0.68.0 (/home/builder/.termux-build/nushell/src/crates/nu-pretty-hex)
   Compiling phf_generator v0.11.0
   Compiling papergrid v0.5.1
   Compiling nu-term-grid v0.68.0 (/home/builder/.termux-build/nushell/src/crates/nu-term-grid)
   Compiling nu-command v0.68.0 (/home/builder/.termux-build/nushell/src/crates/nu-command)
   Compiling pwd v1.4.0
error[E0432]: unresolved imports `libc::endpwent`, `libc::getpwent`, `libc::setpwent`
 --> /home/builder/.cargo/registry/src/github.com-1ecc6299db9ec823/pwd-1.4.0/src/unix.rs:1:20
  |
1 | use libc::{c_char, endpwent, getpwent, getpwnam, getpwuid, getuid, passwd, setpwent};
  |                    ^^^^^^^^  ^^^^^^^^ no `getpwent` in the root            ^^^^^^^^ no `setpwent` in the root
  |                    |
  |                    no `endpwent` in the root
  |
help: a similar name exists in the module
  |
1 | use libc::{c_char, endpwent, getutent, getpwnam, getpwuid, getuid, passwd, setpwent};
  |                              ~~~~~~~~
help: a similar name exists in the module
  |
1 | use libc::{c_char, endpwent, getpwent, getpwnam, getpwuid, getuid, passwd, setutent};
  |                                                                            ~~~~~~~~

For more information about this error, try `rustc --explain E0432`.
error: could not compile `pwd` due to previous error
warning: build failed, waiting for other jobs to finish...
error: failed to compile `nu v0.68.0 (/home/builder/.termux-build/nushell/src)`, intermediate artifacts can be found at `/home/builder/.termux-build/nushell/src/target`
ERROR: failed to build.</pre>
</details>

<hr>
<i>
Above error occured when I last tried to update at 2022-09-07 00:38:41 UTC.<br>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/3004229327">3004229327</a><br><br>
<b>Note:</b> Automatic updates will be disabled until this issue is resolved.<br>
</i>

--- a/packages/nushell/build.sh
+++ b/packages/nushell/build.sh
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://www.nushell.sh
 TERMUX_PKG_DESCRIPTION="A new type of shell operating on structured data"
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="0.67.0"
+TERMUX_PKG_VERSION="0.68.0"
 TERMUX_PKG_SRCURL=https://github.com/nushell/nushell/archive/$TERMUX_PKG_VERSION.tar.gz
-TERMUX_PKG_SHA256=a8c4c092087d035805781dff11000f2b7c4a44ae487b013f9d0a59c4dc610546
+TERMUX_PKG_SHA256=c170ce2a0fa931194c8169585608747fea0cbf7c957f67c6e2fd54c0c5071c64
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_DEPENDS="openssl, zlib"
 TERMUX_PKG_AUTO_UPDATE=true
@@ -31,12 +31,18 @@ termux_step_pre_configure() {
 	: "${CARGO_HOME:=$HOME/.cargo}"
 	export CARGO_HOME
 
+	rm -rf $CARGO_HOME/registry/src/github.com-*/libgit2-sys-*
+	rm -rf $CARGO_HOME/registry/src/github.com-*/pwd-*
 	cargo fetch --target "${CARGO_TARGET_NAME}"
 
 	for d in $CARGO_HOME/registry/src/github.com-*/libgit2-sys-*/libgit2; do
 		patch --silent -p1 -d ${d} \
 			<$TERMUX_SCRIPTDIR/packages/libgit2/src-rand.c.patch || :
 		cp $TERMUX_SCRIPTDIR/packages/libgit2/getloadavg.c ${d}/src/ || :
+	done
+
+	for d in $CARGO_HOME/registry/src/github.com-*/pwd-*; do
+		patch --silent -p1 -d ${d} < $TERMUX_PKG_BUILDER_DIR/crates-pwd-for-android.diff || :
 	done
 
 	mv $TERMUX_PREFIX/lib/libz.so.1{,.tmp}

--- a/packages/nushell/crates-pwd-for-android.diff
+++ b/packages/nushell/crates-pwd-for-android.diff
@@ -1,0 +1,55 @@
+diff -uNr pwd-1.4.0/src/unix.rs pwd-1.4.0.mod/src/unix.rs
+--- pwd-1.4.0/src/unix.rs
++++ pwd-1.4.0.mod/src/unix.rs
+@@ -1,4 +1,6 @@
+-use libc::{c_char, endpwent, getpwent, getpwnam, getpwuid, getuid, passwd, setpwent};
++use libc::{c_char, getpwnam, getpwuid, getuid, passwd};
++#[cfg(not(target_os = "android"))]
++use libc::{endpwent, getpwent, setpwent};
+ use std::ffi::{CStr, CString};
+ 
+ use crate::errors::{PwdError, Result};
+@@ -40,6 +42,7 @@
+ impl Iterator for PasswdIter {
+     type Item = Passwd;
+ 
++    #[cfg(not(target_os = "android"))]
+     fn next(&mut self) -> Option<Self::Item> {
+         if !self.inited {
+             unsafe {
+@@ -63,6 +66,11 @@
+             None
+         }
+     }
++
++    #[cfg(target_os = "android")]
++    fn next(&mut self) -> Option<Self::Item> {
++        None
++    }
+ }
+ 
+ fn cstr_to_string(cstr: *const c_char) -> Result<String> {
+@@ -86,10 +94,19 @@
+             Some(cstr_to_string(pwd.pw_passwd)?)
+         };
+ 
+-        let gecos = if pwd.pw_gecos.is_null() {
+-            None
+-        } else {
+-            Some(cstr_to_string(pwd.pw_gecos)?)
++        let gecos = {
++            #[cfg(not(all(target_os = "android", target_pointer_width = "32")))]
++            {
++                if pwd.pw_gecos.is_null() {
++                    None
++                } else {
++                    Some(cstr_to_string(pwd.pw_gecos)?)
++                }
++            }
++            #[cfg(all(target_os = "android", target_pointer_width = "32"))]
++            {
++                None
++            }
+         };
+ 
+         Ok(Passwd {

--- a/packages/nushell/user_home_dir.patch
+++ b/packages/nushell/user_home_dir.patch
@@ -1,12 +1,26 @@
 --- a/crates/nu-path/src/tilde.rs
 +++ b/crates/nu-path/src/tilde.rs
-@@ -53,6 +53,12 @@
+@@ -1,4 +1,4 @@
+-#[cfg(all(unix, not(target_os = "macos")))]
++#[cfg(all(unix, not(target_os = "macos"), not(target_os = "android")))]
+ use pwd::Passwd;
+ use std::path::{Path, PathBuf};
+ 
+@@ -45,7 +45,7 @@
+     }
+ }
+ 
+-#[cfg(all(unix, not(target_os = "macos")))]
++#[cfg(all(unix, not(target_os = "macos"), not(target_os = "android")))]
+ fn user_home_dir(username: &str) -> PathBuf {
+     let passwd = Passwd::from_name(username);
+     match &passwd.ok() {
+@@ -60,6 +60,11 @@
      // Returns home dir of user.
  }
  
 +#[cfg(target_os = "android")]
-+#[allow(unused)]
-+fn user_home_dir(username: &str) -> PathBuf {
++fn user_home_dir(_username: &str) -> PathBuf {
 +    PathBuf::from(String::from("@TERMUX_HOME@"))
 +}
 +

